### DISCUSSION TITLE
UiBasedCollector

### DISCUSSION THREAD

**Discussion by JaneAtt** [NightscoutFoundation/xDrip#2473] [Q&A]
Directed to post here by Navid Fo for the attention of Jon
xDrip won't reconnect (Signal Missed), using Pixel 6a with Dexcom G6 and CamAPS FX (NB CamAPS FX still talking to G6 Transmitter)
Updated to latest pre-release
![Screenshot_20221117-101137](https://user-images.githubusercontent.com/118431752/202399100-a43b2988-86af-473e-a37b-b619f86d0d73.png)
<Screenshot_20221117-100821.png>
![Screenshot_20221117-100800](https://user-images.githubusercontent.com/118431752/202399536-fe0c4ffe-ef28-42cb-a956-8387c6811859.png)

<Screenshot_20221117-100821.png>
![Screenshot_20221117-100821](https://user-images.githubusercontent.com/118431752/202400299-9df681f7-9ac7-4a6b-ab76-b7a945585a26.png)

**Comment by jamorham** [comment]
Questions:
Did this work before?
Do you have the notification displaying that shows glucose value?
Are you using the Companion Bluetooth feature?

**Reply by JaneAtt** [reply]
Yes it has been working without any problems. 
No glucose values being displayed.
Re Are you using the Companion Bluetooth feature? Do you mean have I selected the 'Companion App' as the Hardware Data Source? If so then yes.
![Screenshot_20221117-203027](https://user-images.githubusercontent.com/118431752/202409771-e5cac842-8841-496c-9b9c-1be6e178f7bf.png)
![Screenshot_20221117-203046](https://user-images.githubusercontent.com/118431752/202409855-95e4cd91-e96a-4b9f-8972-5134c19076e8.png)

**Comment by jamorham** [comment]
To avoid any confusion I'll just deal with one response of yours at a time.

In your notification area if you swipe down do you see the notification from the companion app which has a glucose number in it?

**Reply by JaneAtt** [reply]
Sorry, I am confused!
"In your notification area”…of my phone or is there a notification area in xDrip?
“Companion app” in my case would this be CamAPS?

**Comment by JaneAtt** [comment]
Sorry, I am confused!
"In your notification area”…of my phone or is there a notification area in xDrip?
“Companion app” in my case would this be CamAPS?

On 17 Nov 2022, at 8:42 pm, JamOrHam ***@***.******@***.***>> wrote:

This message was sent from outside the company by someone with a display name matching a user in your organisation. Please do not click links or open attachments unless you recognise the source of this email and know the content is safe.


To avoid any confusion I'll just deal with one response of yours at a time.

In your notification area if you swipe down do you see the notification from the companion app which has a glucose number in it?

**Reply by jamorham** [reply]
Yes, the Companion collector (also called UiBased) works by reading notifications so I'm asking if the notification is showing correctly from the app you are wanting it to collect from.

**Reply by JaneAtt** [reply]
Ok, thanks. 
I have no notifications from CamAPS in phone notifications.

**Reply by JaneAtt** [reply]
These are the notification settings for CamAPS
![Screenshot (17 Nov 2022 9_08_48 pm)](https://user-images.githubusercontent.com/118431752/202418438-e143e2db-d00f-499b-bba5-89e8a48eb3d9.png)

**Comment by JaneAtt** [comment]
I should clarify regarding your question "Did this work before". It worked perfectly when I was using Version e13a76e-2022.08.03. Out of nowhere it stopped working and it was suggested to me (through Facebook Group) to try updating to pre-release. I updated to pre-release however it did not fix it.

**Reply by jamorham** [reply]
Does it work if you enable the info switch shown in your screenshot above?

**Reply by JaneAtt** [reply]
YES!!! Thank you! I did not turn this off intentionally...so unsure how this would happen...?
Thank you for your patience! I clearly am not tech savvy!
Really appreciate your help!

**Reply by jamorham** [reply]
Ok, so that can happen if you hide the notification, but the UI collector normally works anyway in that situation as far as I know so I'm wondering if its something to do with your phone model. Could you let me know which phone model and android version you are using?

**Comment by JaneAtt** [comment]
Google Pixel 6a
And it seems that my phone has updated to Android 13

On 17 Nov 2022, at 9:46 pm, JamOrHam ***@***.******@***.***>> wrote:

This message was sent from outside the company by someone with a display name matching a user in your organisation. Please do not click links or open attachments unless you recognise the source of this email and know the content is safe.


Ok, so that can happen if you hide the notification, but the UI collector normally works anyway in that situation as far as I know so I'm wondering if its something to do with your phone model. Could you let me know which phone model and android version you are using?
